utils/utils.jsx

```diff
@@ -1655,9 +1655,16 @@ export function removePrefixFromLocalStorage(prefix) {
 }
 
 export function copyToClipboard(data) {
+    // Attempt to use the newer clipboard API when possible
```

Comment:
From what I can tell, this isn't actually possible since at least Firefox doesn't support `navigator.clipboard` yet, but it does work when we stub out the method in Cypress tests